frontend/src/Settings/Profiles/Metadata/ReleaseStatusItem.js

```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames';
```

Comment:
I think this was pulled from Readarr as it wasn't needed?

Reply:
Incoming commit